### PR TITLE
PLANET-5934 Define 8px spacers

### PR DESCRIPTION
**Pull request by comzeradd**
Ref: https://jira.greenpeace.org/browse/PLANET-5934

---

Aligning spacing with variables changes in master-theme.

Counterpart of https://github.com/greenpeace/planet4-master-theme/pull/1536

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [venus](https://www-dev.greenpeace.org/test-venus) | [admin](https://www-dev.greenpeace.org/test-venus/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-venus/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-venus) | [composer-local.json](https://github.com/greenpeace/planet4-test-venus/blob/main/composer-local.json)

:watch: 2021.12.13 06:39:14

File: assets/src/styles/blocks/Articles.scss [hunk 1]
```diff
@@ -127,7 +127,7 @@
 
 .article-list-item-bullet {
   display: inline-block;
-  padding: 0 3px 0 $n5;
+  padding: 0 $sp-x 0 $sp-x;
```
Comment (GP-Dan-Tovbein) [inline assets/src/styles/blocks/Articles.scss:130]: Can be shorthanded to `0 $sp-x;`

File: assets/src/styles/blocks/Columns.scss [hunk 2]
```diff
@@ -265,8 +265,8 @@
           display: block;
           cursor: pointer;
           text-decoration: none;
-          padding: $n10 $n10 $n5 $n10;
-          margin-bottom: $space-sm;
+          padding: $sp-1 $sp-1 $sp-x $sp-1;
```
Comment (GP-Dan-Tovbein) [inline assets/src/styles/blocks/Columns.scss:268]: can be shorthanded to `$sp-1 $sp-1 $sp-x;`

File: assets/src/styles/blocks/Columns.scss [hunk 3]
```diff
@@ -303,13 +303,13 @@
...
             margin-bottom: 0;
-            margin-top: $space-lg;
+            margin-top: $sp-4;
             width: 100%;
           }
         }
 
         .card-block {
-          padding: calc(#{$space-lg} - #{$space-md}) 28px $space-lg 28px;
+          padding: calc(#{$sp-4} - #{$sp-2}) 28px $sp-4 28px;
```
Comment (GP-Dan-Tovbein) [inline assets/src/styles/blocks/Columns.scss:312]: Can be shorthanded to `calc(#{$sp-4} - #{$sp-2}) 28px $sp-4;`

File: assets/src/styles/blocks/ENForm/components/_enform-side-style.scss [hunk 4]
```diff
@@ -83,11 +83,11 @@
       max-width: 444px;
       box-sizing: content-box;
       box-shadow: 0 2px 5px rgba(0, 0, 0, .25);
-      padding: $space-xl 30px 34px 30px;
+      padding: $sp-6 30px 34px 30px;
```
Comment (GP-Dan-Tovbein) [inline assets/src/styles/blocks/ENForm/components/_enform-side-style.scss:86]: Can be shorthanded to `$sp-6 30px 34px;`

File: assets/src/styles/blocks/OldENForm/components/_enform-side-style.scss [hunk 5]
```diff
@@ -83,11 +83,11 @@
       max-width: 444px;
       box-sizing: content-box;
       box-shadow: 0 2px 5px rgba(0, 0, 0, .25);
-      padding: $space-xl 30px 34px 30px;
+      padding: $sp-6 30px 34px 30px;
```
Comment (GP-Dan-Tovbein) [inline assets/src/styles/blocks/OldENForm/components/_enform-side-style.scss:86]: Can be shorthanded to `$sp-6 30px 34px;`

**Review by GP-Dan-Tovbein — COMMENTED** [review]
I've left some suggestion but code looks to me!

**Comment by comzeradd** [comment]
@GP-Dan-Tovbein PR updated

**Review by GP-Dan-Tovbein — APPROVED** [review]
Code look good to me!

**Review by GP-Dan-Tovbein — APPROVED** [review]
Code still looks good to me!

**Pull request merged**